### ISSUE TITLE
Improve GITFans?

### ISSUE THREAD

**Issue by fingolfin** [oscar-system/Oscar.jl#2036]
First off, our test for them are too slow. They take over 2 minutes in our CI runs. Perhaps a smaller test example can be used?

Some timings from [this GitHub Actions log](https://github.com/oscar-system/Oscar.jl/actions/runs/4375701490)

|                                                    Filename | Runtime in s | + Compilation | + Recompilation | Allocations in MB |
|-------------------------------------------------------------|--------------|---------------|-----------------|-------------------|
|                              ./Experimental/GITFans-test.jl |      128.782 |       14.0616 |             0.0 |            909.09 |

Secondly, this code still uses raw polymake objects. As such it is not a great example for OSCAR anymore. It should be rewritten to use OSCAR polytopes/cones/etc. (or if any of those are missing, then of course we should request that those be added)

**Comment by jankoboehm** [comment]
We are already busy with that. Most of the functionality is already there. One point which is still missing to do the update are fans with a group action, which are existent in polymake, but not in OSCAR, @lkastner is already on that topic. We can do a first round of updates with the other polyhedral stuff already available, also replacing the example with an easier one, rest when back from holidays. The code should then also move out of experimental.

**Comment by lkastner** [comment]
Yes, once @jankoboehm opens a PR for the polyhedral things, I will push a suitable constructor. At least that is what I remember we agreed on. In case you want proceed differently, please let me know.

**Comment by lkastner** [comment]
I have made #2100 with a suitable constructor. The rest I already went over with @jankoboehm last week. Let me know if there are further questions or other necessary modifications.

**Comment by benlorenz** [comment]
The timing is significantly better now:
### Stats per file

|                                                        Filename | Runtime in s | + Compilation | + Recompilation | Allocations in MB |
|-----------------------------------------------------------------|--------------|---------------|-----------------|-------------------|
|                           experimental/GITFans/test/runtests.jl |        27.85 |          7.49 |            0.00 |             950.4 |

**Comment by lkastner** [comment]
There is no raw polymake usage left in gitfans.
